### ISSUE TITLE
Training the model with custom data

### ISSUE THREAD

**Issue by upstream-team** [Thomasbehan/SkinVestigatorAI#27]
I have a question regarding machine learning. I am wondering if it is possible to train a system using custom data to accurately detect and classify skin problems such as rashes and acne. I am specifically interested in identifying the type of rashes and acne. I would appreciate any insights or guidance on how to approach this. Thank you!

**Comment by Thomasbehan** [comment]
Absolutely, you can extend the capability of SkinvestigatorAI to detect and classify skin problems like rashes and acne. Just add your images to the /data directory (if there isn't one you can create one). Organize them in test, train, and validation subfolders. Within each of these subfolders, create a separate folder for each diagnosis you want the AI to detect. For instance: /data/validation/melanoma/image.png. or /data/train/eczema/image.jpg
You can store as many images in each diagnoses folder as you like.

Once your data is organized, you can train the AI by running the training script: /skinvestigatorai/core/ai/train.py.

Hope this helps.
